frontend/src/pages/dashboard/components/tables/drivers-table/columns/columns.tsx

```diff
@@ -11,6 +11,22 @@ const columns: ColumnDef<DriverWithUserData>[] = [
     footer: 'Full name',
     size: 250,
   },
+  {
+    header: 'Photo',
+    accessorFn: (driver: DriverWithUserData) =>
+      driver.driver.avatarUrl ??
+      'https://forwardsummit.ca/wp-content/uploads/2019/01/avatar-default.png',
```

Comment:
I saw DEFAULT_AVATAR - let's put url there. Also, let's upload the default avatar to our s3 bucket.

Reply:
Yea, this is expected to be added after merging @TetianaDoroshko pr with orders